Updating the configuration settings

kennytonychakola opened the pull request:
To incorporate the changes to the configuration file as a result of the Symfony 4.4 update.

kennytonychakola committed:
Updating the configuration settings

To incorporate the changes to the configuration file as a result of the Symfony 4.4 update.

kennytonychakola committed:
Documentation changes to installation steps.

To incorporate the changes to the installation steps due to Symfony 4.4 update.

ubermichael commented on source/config.rst:
```diff
@@ -3,48 +3,34 @@
 
-     # number of posts to show on the home page
-     nines_blog.homepage_posts: 3
+.. code-block:: yaml
```
This is not yaml, and shouldn't be highlighted like yaml. You could call it bash - the syntax is the same even though it isn't a bash file.

ubermichael commented on source/config.rst:
```diff
@@ -3,48 +3,34 @@
-     nines_blog.homepage_posts: 3
+.. code-block:: yaml
+   
+   parameters:
```
The line `parameters:` should not appear in this file.

ubermichael commented on source/install.rst:
```diff
@@ -10,7 +10,7 @@ Installation
 Make sure the requirements are satisfied.
 
-The BTD application is based on Symfony 3.2. Installation follows the normal
+The PI application is based on Symfony 4.4. Installation follows the normal
```
Good catch!

ubermichael commented on source/install.rst:
```diff
@@ -60,7 +60,7 @@ Sometimes composer runs out of memory. If that happens, try this alternate.
 .. code-block:: bash
 
-  mysql -u pi -p pi < pi-2020-01-22-dhil-r30.sql
+  mysql -u pi -p pi < pi-2020-03-25-dhil-r35.sql
```
It would probably be useful to explain the format here and tell people that the name of the file will probably be different in the future.

ubermichael commented on source/install.rst:
```diff
@@ -76,20 +76,33 @@ If you don't have the data, then proceed to step 8 and 9
 .. code-block:: bash
 
-  ./bin/console fos:user:create --super-admin  
+  ./bin/console nines:create:user --super-admin  
```
`--super-admin` is not an option for `nines:create:user`.

ubermichael commented on source/install.rst:
```diff
@@ -76,20 +76,33 @@ If you don't have the data, then proceed to step 8 and 9
+
+12. Please follow the instructions in the config.rst file to set up the configuration settings for this project.
+
+13. Configure the web server. The application's `web/` directory must
```
In Symfony 4.4 the `public/` directory must be accessible.

ubermichael commented on source/install.rst:
```diff
@@ -76,20 +76,33 @@ If you don't have the data, then proceed to step 8 and 9
   
+.. code-block:: bash
+
+  symfony server:start
```
It's a good idea to include directions for stopping the server once it is up and running.

ubermichael reviewed (changes requested):
This looks really good although there are a few minor changes necessary.

kennytonychakola committed:
Changes to the Configurtion file in the documentation.

To incorporate the changes to the configuration file as a result of the Symfony 4.4 update.

kennytonychakola committed:
Document changes to the Installation steps.

To incorporate the changes to the installation steps due to Symfony 4.4 update.
